src/api/app/models/staging/staging_project.rb

```diff
@@ -100,6 +104,14 @@ def unassign_managers_group(managers)
       relationships.find_by(group: managers, role: role).try(:destroy!)
     end
 
+    def current_checks
+      relevant_checks + relevant_checks_for_architectures
+    end
+
+    def current_missing_checks
+      (relevant_status_reports + relevant_status_reports_for_architectures).map(&:missing_checks).flatten
```

Comment:
@ChrisBr do you have a better idea for missing_checks?

Reply:
missing checks aren't in the database, they are calculcated. So I don't think there is a way

Reply:
Does this work when there is no relevant status report?

Reply:
But anyway, we can fix this in a follow up / when we merge the PR from @krauselukas. Would prefer to get this in ASAP.

Reply:
me too - as there is also #6261 pending also affected by this.

Reply:
> Does this work when there is no relevant status report?

Good you ask, because it won't. That's why we created current_status_report on checkables. But with the current coverage, it's pretty easy to mess this up :(

But as already written: let's fix afterwards.